tests/playwright/sanity/container.test.ts

```diff
@@ -158,10 +158,7 @@ test.describe( 'Container tests @container', () => {
 
 		// Assert
 		// Take screenshot.
-		expect.soft( await pageView.screenshot( {
-			type: 'jpeg',
-			quality: 90,
-		} ) ).toMatchSnapshot( 'heading-boxed-absolute.jpeg' );
+		await expect( pageView ).toHaveScreenshot( 'heading-boxed-absolute.png' );
```

Comment:
I saw this in a few places.
I thought we that our convention was that all screenshot tests have soft assertions.
```suggestion
		await expect.soft( pageView ).toHaveScreenshot( 'heading-boxed-absolute.png' );
```

WDYT?